chapters/0-introduction.tex

```diff
@@ -0,0 +1,266 @@
+is a key skill for researchers and their staff.
+At the same time, the scope and scale of empirical research projects is expanding:
+more people are working on the same data over longer timeframes.
+As the ambition of development researchers grows, so too has the complexity of the data
+on which they rely to make policy-relevant research conclusions.
+Yet there are few guides to the conventions, standards, and best practices
+that are fast becoming a necessity for empirical research.
+
+This book aims to fill that gap.
+It covers data workflows at all stages of the research process,
+from design to data acquisition and analysis.
+Its content is not sector-specific;
+it will not teach you econometrics,
+or how to design an impact evaluation.
+There are many excellent existing resources on those topics.
+Instead, this book will teach you how to think about all aspects of your research from a data perspective,
+how to structure research projects to maximize data quality,
+and how to institute transparent and reproducible workflows.
+The central premise of this book is that data work is a ``social process'',
+in which many people on a team need to have the same ideas
+about what is to be done, and when and where and by whom,
+so that they can collaborate effectively on large, long-term research projects.
+
+\textit{Development Research in Practice} compiles the ideas, best practices and software tools
+that the DIME Analytics team
+has developed while supporting DIME's global impact evaluation portfolio.
+Each chapter in this book focuses on one task, providing a primarily narrative account of:
+what you will be doing; where in the workflow this task falls;
+when it should be done; and how to implement it according to best practices.
+We will use broad terminology throughout this book to refer to some typical research team members:
+\textbf{principal investigators (PIs)} who are responsible for
+the overall design and stewardship of the study;
+\textbf{field coordinators (FCs)} who are responsible for
+the implementation of the study on the ground;
+and \textbf{research assistants (RAs)} who are responsible for
+handling data processing and analytical tasks.
+
+We will not always give a lot of highly specific implementation details in this text,
+but will often point you to where they can be found on the \textbf{DIME Wiki}.\sidenote{Like this:
+\url{https://dimewiki.worldbank.org/Primary_Data_Collection}}
+The DIME Wiki is one of DIME Analytics' flagship products,
+a free online collection of our resources and best practices.\sidenote{
+\url{https://dimewiki.worldbank.org}}
+This book complements the DIME Wiki by providing a structured narrative
+of the data workflow for a typical research project.
+The Wiki, by contrast, provides unstructured but detailed information
+on how to complete each task, and links to further practical resources.
+
+\section{Adopting reproducible analytical practices through code}
```

Comment:
I think this section is where we want to address Arianna's comment that "The introduction should have a clear motivation of why data and not only research design is important for generating credible evidence. Here you need to make reference to actual research on the perils of poor data on research conclusions." that's the only major comment of hers that i see as still pending.